app/style/themes/darkTheme.js

```diff
@@ -218,8 +218,14 @@ const darkTheme = {
   "menu-trezor-active": "url('style/icons/trezorActiveDark.svg')",
   "menu-trezor-default": "url('style/icons/trezorDefaultDark.svg')",
   "menu-trezor-hover": "url('style/icons/trezorHoverDark.svg')",
-  "menu-cancel-rescan-icon": "url('style/icons/menuCancelRescanDark.svg')"
-
+  "menu-cancel-rescan-icon": "url('style/icons/menuCancelRescanDark.svg')",
```

Comment:
Please move these to `themes/icons.js` => it's the shared icons file, here we specify dark specific icons, same hold for icons in `lightTheme.js`